devops/sql/creditApplication.sql

```diff
@@ -0,0 +1,399 @@
+(62, 2, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(63, 3, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(64, 4, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(65, 5, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(66, 6, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(67, 7, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(68, 8, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(69, 9, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(70, 10, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(71, 11, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(72, 12, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(73, 13, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(74, 14, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(75, 15, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(76, 16, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(77, 17, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(78, 18, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(79, 19, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(80, 20, 'Payslip3', 'Payslip for month 3', 's3://core-banking-sample-data/Document samples/Payslip.png', 'Payslip', '2021-03-08', null, false),
+(81, 1, 'Meldezettle', 'Address proff', 's3://core-banking-sample-data/Document samples/meldezettle.png', 'Meldezettle', '2021-03-08', null, false),
```

Comment:
it is Meldezettel, not Meldezettle, could you please fix it?

Reply:
Updated the names